tests/e2e/react-component/package.json

```diff
@@ -2,8 +2,14 @@
   "name": "react-component-e2e",
   "version": "1.0.0",
   "private": true,
+  "scripts": {
+    "dev:bundle": "../../node_modules/.bin/rsbuild dev --environment=bundle",
+    "dev:bundle-false": "../../node_modules/.bin/rsbuild dev --environment=bundleFalse",
+    "dev:umd": "../../node_modules/.bin/rsbuild dev --environment=umd"
```

Comment:
why not add rsbuild to devdeps in this package.json

Reply:
It's using @rsbuild/core in `../../` to start by the test runner.

Reply:
using relative path to call bin file is a bit ugly

Reply:
Could refactor in other PR.